package.json

```diff
@@ -100,11 +101,11 @@
     "@chanzuckerberg/eslint-plugin-stories": "^3.2.10",
     "@chanzuckerberg/prettier-config-edu": "^1.0.2",
     "@chanzuckerberg/story-utils": "^3.0.11",
-    "@commitlint/cli": "^16.3.0",
-    "@commitlint/config-conventional": "^16.2.4",
-    "@commitlint/prompt-cli": "^16.3.0",
+    "@commitlint/cli": "^17.4.2",
+    "@commitlint/config-conventional": "^17.4.2",
+    "@commitlint/prompt-cli": "^17.4.2",
     "@geometricpanda/storybook-addon-badges": "^0.2.2",
-    "@size-limit/preset-small-lib": "^4.12.0",
+    "@size-limit/file": "^8.1.2",
```

Comment:
Replaced @size-limit/preset-small-lib with @size-limit/file.

The "small-lib" preset uses webpack to do its thing. Based on our config we clearly want to check the result of our own `build` command, and can use @size-limit/file directly.